spec/index.html

```diff
@@ -1214,9 +1160,9 @@ <h2>Best Practices</h2>
 
       <aside class="example" title="Convenience Context">
         <p>Code of the <a>convenience context</a>. Available as:
-          <a href="https://yaml-ld.dev/context">https://yaml-ld.dev/context</a>.
+          <a href="https://github.com/json-ld/convenience-context">https://github.com/json-ld/convenience-context</a>.
```

Comment:
TODO: replace with json-ld.github.io/… 

Where is the context, in fact?

Reply:
Should be at `https://json-ld.org/contexts/yaml-ld.jsonld` or similar.

Reply:
I have submitted a PR to json-ld.org for that.